README.md

```diff
@@ -1 +1,19 @@
-# qiita-paper
\ No newline at end of file
+# Reproducing Figure 2
+
+Figure 2A-C were fully generated in Qiita, to get the same coloring and shapes that were used in the manuscript, you can access each of the individual plots and load the Emperor settings directly. The main analysis can be found within [Qiita](https://qiita.ucsd.edu/analysis/description/15093/), you must be log in to Qiita to be able to access it.
+
+Figure 2A was generated using the artifact [`Figure 2A (ID: 43637)`](https://qiita.ucsd.edu/artifact/html_summary/ordination_results/43637/index.html), to regenerate the coloring scheme you can load the file `emperor-settings-43637.json` within this repository.
+
+Figure 2B was generated using the artifact [`Figure 2B (ID: 43666)`](https://qiita.ucsd.edu/artifact/html_summary/ordination_results/43666/index.html), to regenerate the coloring scheme you can load the file `emperor-settings-43666.json` within this repository.
+
+Figure 2C was generated using the artifact [`Figure 2C (ID: 43729)`](https://qiita.ucsd.edu/artifact/html_summary/ordination_results/43729/index.html), to regenerate the coloring scheme you can load the file `emperor-settings-43729.json` within this repository.
+
+## Reproducing Figure 2D
+
+To open the IPython notebook you will need python3 and these packages:
```

Comment:
IPython -> Jupyter, possible to do this replacement everywhere else if/as appropriate?

Reply:
Also, python3 -> Python 3.5, (assuming that is the version)